requirements.txt

```diff
@@ -1,2 +1,3 @@
 Appium-Python-Client==2.6.1;python_version >= '3.0'
-browserstack-local==1.2.2
\ No newline at end of file
+browserstack-local==1.2.2
```

Comment:
unpin the version for this as well will pick latest (there are changes in later binary which are needed)

Reply:
For Appium-Python-Client? Also as browserstack-local is not required, should I remove that?

Reply:
Appium-Python-Client is > 3 so it's fine will always pick latest as we are only specifying minimum
yes can remove local

Reply:
done